app/assets/javascripts/discourse/components/share-panel.js

```diff
@@ -43,31 +42,12 @@ export default Component.extend({
 
   didInsertElement() {
     this._super(...arguments);
-
-    const shareUrl = this.shareUrl;
-    const $linkInput = $(this.element.querySelector(".topic-share-url"));
-    const $linkForTouch = $(
-      this.element.querySelector(".topic-share-url-for-touch a")
-    );
-
-    schedule("afterRender", () => {
-      if (!this.capabilities.touch) {
-        $linkForTouch.parent().remove();
-
-        $linkInput
-          .val(shareUrl)
-          .select()
-          .focus();
-      } else {
-        $linkInput.remove();
-
-        $linkForTouch.attr("href", shareUrl).text(shareUrl);
-
-        const range = window.document.createRange();
-        range.selectNode($linkForTouch[0]);
-        window.getSelection().addRange(range);
-      }
-    });
+    window.setTimeout(() => {
```

Comment:
setInterval/setTimeout should never be used, we still have some in the codebase I think... but prefer `Ember.run.later`